### ISSUE TITLE
Browserify Cannot find Template Modules in Dist

### ISSUE THREAD

**Issue by lloy0076** [angular-ui/bootstrap#5591]
### Bug description:

Using Laravel Elixir "gulp browserify" I get:

```
[19:59:22] gulp-notify: [Laravel Elixir] Browserify Failed!: Cannot find module './dist/ui-bootstrap-tpls'
```

...when my "app.js" requires angular-ui-bootstrap.

### Solution

I made a symlink like this:

```
ln -s ui-bootstrap-tpls-1.2.3.js ui-bootstrap-tpls.js
```

...and browserify stopped complaining.

### Versions etc

package.json:

```
{
  "private": true,
  "devDependencies": {
    "gulp": "^3.8.8",
    "info": "~1.0.0",
    "laravel-elixir-browserify": "~0.8.1"
  },
  "dependencies": {
    "laravel-elixir": "^4.0.0",
    "bootstrap-sass": "^3.0.0",
    "jquery": "~2.2.0",
    "babel-preset-es2015": "~6.5.0",
    "babel-preset-react": "~6.5.0",
    "angular-ui-router": "~0.2.18",
    "api-check": "~7.5.5",
    "angular": "~1.5.0",
    "angular-formly": "~8.0.0",
    "bootstrap": "~3.3.6",
    "angular-formly-templates-bootstrap": "~6.3.0",
    "angular-route": "~1.5.0",
    "lodash": ">=4.0.0",
    "angular-smart-table": "~2.1.6",
    "angular-ui-bootstrap": "~1.2.2-SNAPSHOT",
    "angular-touch": "~1.5.0",
    "angular-animate": "~1.5.0",
    "restangular": "~1.5.2",
    "angular-utils-pagination": "~0.11.0"
  }
}
```

Minimal JavaScript for to get Browserify to trigger (I put this in resources/assets/js/bomb.js relative to the package.json/composer.json):

```
(function() {

'use strict';

window.$ = window.jQuery = require('jquery')
require('angular-ui-bootstrap');
}());
```

The gulpfile.js:

```
var elixir = require('laravel-elixir');

/*
 * Fix for default precision.
 *
 * See http://tinyurl.com/h8mnjfd.
 */
elixir.config.css.sass.pluginOptions.precision = 10;

elixir(function(mix) {
    mix.sass('app.scss')
       //.browserify('angular.js')
       .browserify('bomb.js')
       .version(
          ['js/bomb.js',
           'css/app.css']
    );
});
```

That's from Laravel 5.2 (a zip of the composer.lock is attached).

[composer.zip](https://github.com/angular-ui/bootstrap/files/160249/composer.zip)

**Comment by ravshansbox** [comment]
ui-bootstrap-tpls.js file incorrectly required inside index.js.

**Comment by artaommahe** [comment]
same error with 1.10 and 1.2.3 ui-bootstrap versions

**Comment by philipstanislaus** [comment]
+1, same problem in webpack.

**Comment by wesleycho** [comment]
I see...I forgot to rename the files - will do another release shortly.

**Comment by wesleycho** [comment]
Released 1.2.4, should fix this.
